src/UI/Http/Rest/Controller/Event/GetEventsController.php

```diff
@@ -26,8 +26,8 @@ class GetEventsController extends QueryController
      */
     public function __invoke(Request $request): JsonResponse
     {
-        $page = $request->get('page', 1);
-        $limit = $request->get('limit', 50);
+        $page = (int) $request->get('page', 1);
```

Comment:
That about also add `Assertion::integer` too?

Reply:
I'm not sure where can I do that. Query parameters are always strings. :( Or maybe I am wrong?

Reply:
It's now about casting its about non numbers request. Like `limit=kk`.

I prefer add this assertion at the top of the function

Reply:
Ok. https://github.com/jorge07/symfony-4-es-cqrs-boilerplate/pull/50/commits/c9d051f16cc4a5c1c6459e3aeef25271dc98f386 It is okay for you? Now client is forced to use body parameters instead of query in `GET`.